MekHQ/src/mekhq/campaign/parts/ProtomekLocation.java

```diff
@@ -165,7 +163,7 @@ public boolean forQuad() {
     public boolean isSamePartType(Part part) {
         return part instanceof ProtomekLocation
                 && getLoc() == ((ProtomekLocation)part).getLoc()
-                && getUnitTonnage() == ((ProtomekLocation)part).getUnitTonnage()
+                && getUnitTonnage() == part.getUnitTonnage()
```

Comment:
Double check that ProtomekLocation doesn't have an implementation of getUnitTonnage()

Reply:
Same as above